src/main/java/com/cloudbees/jenkins/plugins/docker_build_env/Docker.java

```diff
@@ -294,6 +294,11 @@ private String getDocker0Ip(Launcher launcher, String image) throws IOException,
 
         String route = out.toString("UTF-8").trim();
 
+        // ensures that the the default IP exists so that the return is not null
+        if (route.indexOf("default") ==  -1){
```

Comment:
You are right here, it is more precise and will probably work faster(maybe?).

Change is done